### ISSUE TITLE
[0.9.0.3 br-78] Craft Resource Multiplier applies inconsistently, or something else is up

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoIssues#18594]
I've decided to play some solo ECO with a Craft Resource Multiplier of 0.05. Most of the time, it works fine - I can make 20x amount of items than I normally would. However, there are some areas it applies inconsistently.

In Campfire, a lot of recipes have yellow border for inputs, meaning the multiplier doesn't kick in:

![image](https://user-images.githubusercontent.com/964559/95150202-8d00bc80-073c-11eb-8569-019bc054baca.png)

Which is fair enough, it's the same with Upgrade Modules. However, the Workbench works differently:

![image](https://user-images.githubusercontent.com/964559/95150275-b1f52f80-073c-11eb-8d44-8ebb3b8acc00.png)

Despite not taking Upgrade Modules and having inputs in yellow border, the Craft Resource Multiplier does apply.

This means either the Craft Resource Multiplier applies oddly, or the Workbench recipes should have a green border. This wouldn't change anything during normal play, but would keep things consistent for servers that do use CRM.

**Comment by ThePiachu** [comment]
Yeah, I did notice it can get exponential at times. I wasn't complaining - turning a shovelful of crushed gold into a stockpile of gold was fun :D . But agreed - the resource multiplier should only apply to raw, unprocessed ingredients - rocks, raw food, animal carcasses, raw wood, etc. This way the multiplier will be applied only once and not go crazy the deeper down the tech tree you go.

I guess the only exception would be rock crushing, since you want the ratio to be 20:5, so you can apply the multiplier to crushed rock and crushed ore instead. Nothing bad there.

Also also, I noticed something interesting - the multiplier acts like the Upgrade Modules, as in if you have 10x multiplier on rocking ore concentrate, you will get 10x less tailings than you would if you just multiplied the inputs. I'm not complaining, but that's probably a balance bug too that will make things a bit more complicated.

But other than that, the multiplier works pretty sanely for me so far - it didn't multiply things like Upgrade Modules and the like, and did make things like stockpiles and storage chests proportionally cheaper, so that's good.

**Comment by ThePiachu** [comment]
Maybe if there is a Multiplier being used in the game, we could put a little star on the input that's being affected so people would recognise what is being affected rather than wonder why something takes 20 nails but only 1 log or something

**Comment by ThePiachu** [comment]
Also also worth looking into - Resource Multiplier and Oil stuff. At the moment those don't appear to be affected by it. Might be worthwhile to make things both take less oil barrels to make and produce less empty barrels to keep the ratio proper.

**Comment by IcedForge** [comment]
To clarify on the workbench, it has a yellow border because its static in its behavior that it does not receive any bonus at all ( as there is no professions tied to the crafting of those hewn logs meaning it wont get the 5% bonus from logging) but its not an actual "static" recipe itself so its in a bit of a in between place.
